src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php

```diff
@@ -373,7 +373,7 @@ protected function createNotFoundException($message = 'Not Found', \Exception $p
     protected function createAccessDeniedException($message = 'Access Denied.', \Exception $previous = null)
     {
         if (!class_exists(AccessDeniedException::class)) {
-            throw new \LogicException('You can not use the "createAccessDeniedException" method if the Security component is not available. Try running "composer require security".');
+            throw new \LogicException('You can not use the "createAccessDeniedException" method if the Security component is not available. Try running "composer require symfony/security-bundle".');
```

Comment:
Actually only security-core is required here, maybe we should be specific now that we don't use flex aliases

Reply:
I think in the trait it's OK to suggest the bundle, nobody needs only the exception